aiida/backends/sqlalchemy/migrations/versions/1de112340b16_django_parity_1.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+# pylint: disable=invalid-name,no-member
+"""Parity with Django backend (rev: 0048),
+part 1: Ensure fields to make non-nullable are not currently null
+
+Revision ID: 2b40c8131fe0
+Revises: 535039300e4a
+Create Date: 2021-08-24 18:52:45.882712
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+
+from aiida.common import timezone
+from aiida.common.utils import get_new_uuid
+
+# revision identifiers, used by Alembic.
+revision = '1de112340b16'
+down_revision = '34a831f4286d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():  # pylint: disable=too-many-statements
+    """Convert null values to default values"""
+    db_dbauthinfo = sa.sql.table(
+        'db_dbauthinfo',
+        # sa.Column('aiidauser_id', sa.Integer),
+        # sa.Column('dbcomputer_id', sa.Integer),
```

Comment:
```suggestion
```

Reply:
I would just note here for completeness that, since all the foreign field columns were nullable, in theory there could exist a database where these are null and then the change to `nullable=False` would fail.
But obviously (a) in practice I can't see this ever being the case, since it would already break aiida, and (b) there is no possible default to change these to

Reply:
But so are you suggesting to keeping these commented out lines to remember that these _may_ cause problems? If so, then I would add a comment explaining. Otherwise, I would just remove these comments out lines everywhere. Actually, I think the best solution is to summarize this problem once in the docstring of the migration module saying there are fields that could be null because of the missing explicit `nullable=False`, but that it is unlikely and anyway we couldn't provide a default. And then just delete the commented out lines